src/services/dashboards/resource.js

```diff
@@ -1604,3 +2037,114 @@ export async function resourceDashboard(scopes) {
     domainList: generateResourceDomainList(data, true),
   };
 }
+
+export async function rollUpResourceUse(data) {
+  const rolledUpResourceUse = data.resourceUseResult.reduce((accumulator, resource) => {
+    const exists = accumulator.find((r) => r.url === resource.url);
+    if (!exists) {
+      // Add a property with the resource's URL.
+      return [
+        ...accumulator,
+        {
+          heading: resource.url,
+          url: resource.url,
+          title: resource.title,
+          sortBy: resource.title || resource.url,
+          total: resource.totalCount,
+          isUrl: true,
+          data: [{ title: resource.rollUpDate, value: resource.resourceCount }],
+        },
+      ];
+    }
+
+    // Add the resource to the accumulator.
+    exists.data.push({ title: resource.rollUpDate, value: resource.resourceCount });
+    return accumulator;
+  }, []);
+
+  // Loop through the rolled up resources and add a total.
+  rolledUpResourceUse.forEach((resource) => {
+    resource.data.push({ title: 'Total', value: resource.total });
+  });
+
+  // Sort by total and name or url.
+  // rolledUpResourceUse.sort((r1, r2) => r2.total - r1.total || r1.sortBy.localeCompare(r2.sortBy));
+  rolledUpResourceUse.sort((r1, r2) => r2.total - r1.total || r1.url.localeCompare(r2.url));
```

Comment:
its not sorting by name

Reply:
For now no, the commented out code is the one we will ultimately use however to match closer to your order ( in the case of top 10) we will just use url for now.